pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -856,6 +858,19 @@ public ApnsPayloadBuilder setTimestamp(final Instant timestamp) {
         return this;
     }
 
+    /**
+     * <p>Sets the flag to wake up the app and receive a push token to send updates to the started Live Activity.</p>
```

Comment:
A couple comments here:

1. If this is a single paragraph, I don't think we need `<p>` tags.
2. While this isn't incorrect, I do think there's more of the story we can and should tell for callers. In particular: this is for establishing a new channel for live activities, and it's intended for iOS 18 and newer. If it's helpful to you, I'd be happy to suggest specific wording!

Reply:
Sorry for the delay! I have updated the Javadoc to provide a bit more info. If the changes are fine, otherwise I'm open for any suggestions. In general, I don't like rehashing info from the source (Apple docs in this case), since from the outside most of the time it feels like trying to documenting undocumented behavior.